db/schema.rb

```diff
@@ -259,6 +262,9 @@
     t.integer "region_code"
     t.datetime "created_at", default: -> { "timezone('utc'::text, now())" }, null: false
     t.datetime "updated_at", default: -> { "timezone('utc'::text, now())" }, null: false
+    t.float "latitude"
```

Comment:
should these be `null: true` otherwise its zeroed out

Reply:
@defong - by default existing Sites and Provider will be given a value of nil for lat and long once the migration is run. Geocoder provides a rake task to assign geolocation data but this won't be done in this ticket

```
=> #<Provider:0x00007fd0b497d290
 id: 12,
 address4: nil,
 provider_name: "ACME SCITT 9",
 scheme_member: "is_a_UCAS_ITT_member",
 contact_name: nil,
 year_code: "2020",
 provider_code: "A9",
 provider_type: nil,
 postcode: nil,
 website: nil,
 address1: nil,
 address2: nil,
 address3: nil,
 address3: nil,
 email: "wes_kshlerin@graham.com",
 telephone: "0829 900 3856",
 region_code: nil,
 created_at: Fri, 22 Nov 2019 14:03:01 UTC +00:00,
 updated_at: Fri, 22 Nov 2019 14:03:01 UTC +00:00,
 accrediting_provider: nil,
 last_published_at: nil,
 changed_at: Fri, 22 Nov 2019 14:03:01 UTC +00:00,
 recruitment_cycle_id: 1,
 discarded_at: nil,
 train_with_us: nil,
 train_with_disability: nil,
 accrediting_provider_enrichments: nil,
 latitude: nil,
 longitude: nil>
```